src/components/Input/DefaultInput/DefaultInput.css

```diff
@@ -44,3 +99,38 @@
   line-height: var(--input-large-line-height);
   padding: var(--input-large-side-padding);
 }
+
+.label {
+  font-size: var(--input-label-font-size);
+  font-weight: var(--input-label-font-weight);
+  line-height: var(--input-label-line-height);
+  color: var(--input-label-color);
+  display: flex;
+  width: 100%;
+}
+
+.span {
```

Comment:
please avoid creating classes without a context. A `span` doesn't mean anything other than an HTML tag. A better naming could be 'description'